src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ CHANGELOG
  * Add support for multiple fields containing nested constraints in `Composite` constraints
  * Add the `stopOnFirstError` option to the `Unique` constraint to validate all elements
  * Add support for closures in the `When` constraint
+ * Add method `assertViolation` to `ConstraintValidatorTestCase`
```

Comment:
this changelog entry looks wrong to me, as this method was already there before this PR.